x/subscription/keeper/subscription.go

```diff
@@ -251,3 +251,42 @@ func (k Keeper) CreateSubscription(
 
 	return nil
 }
+
+func (k Keeper) AddProjectToSubscription(ctx sdk.Context, subscriptionOwner string, projectAdmin string, projectName string, enabled bool, projectDescription string, geolocation uint64, vrfpk string) error {
+	sub, found := k.GetSubscription(ctx, subscriptionOwner)
+	if !found {
+		return sdkerrors.ErrKeyNotFound.Wrapf("can't get subscription of %s", subscriptionOwner)
+	}
+
+	plan, found := k.plansKeeper.GetPlan(ctx, sub.GetPlanIndex())
```

Comment:
instead of GetPlan you should use findPlan(block = sub.blockBought)

Reply:
done

Reply:
not done